docs/prover/cli/options.md

```diff
@@ -819,3 +824,9 @@ splits generated here is equal to `2^n` where `n` is the initial splitting depth
 (unless the program has less than `n` branchings, which will be rare in
 practice).
 
+Using Conf Files 
+----------------
+For larger projects, the command line for running the Certora Prover can become large 
```

Comment:
I would probably move this paragraph right into the intro, since it is a general recommendation.  Otherwise I worry it will get lost here.

Reply:
done